src/hls-adapter.js

```diff
@@ -524,7 +572,7 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
    */
   selectTextTrack(textTrack: TextTrack): void {
     if (textTrack instanceof TextTrack && !textTrack.active) {
-      if (this._hls.subtitleTracks.length) {
+      if (this._hls && this._hls.subtitleTracks.length) {
```

Comment:
this is incorrect, if there's no this._hls you need to do nothing, not to go to the else clause